orchestrator/dags/general_data_import.py

```diff
@@ -70,22 +70,22 @@ def download_general_ztm_data(ti) -> None:
         
 def extract_timetable_lines(ti) -> None:
     general_file_name = ti.xcom_pull(key='general_file_name')
-    with open(f'{out_dir}{general_file_name}', "rt", encoding="utf8") as file:
-        for line in file:
+    with open(f'{out_dir}{general_file_name}', "rt", encoding="utf8") as infile:
```

Comment:
@mgierm correct me if I'm wrong, but wouldn't be better to have file_in / input_file instead of infile ?

Reply:
Yeah, maybe "file_in" sounds better than "infile", so @Maryl12342 if you want, you can update these naming, but I won't oppose if you don't want.